pkg/memory-monitor/build.yml

```diff
@@ -0,0 +1,18 @@
+image: eve-memory-monitor
+org: lfedge
+config:
+  binds:
+    - /persist:/persist # for the working directory and output directory
+    - /dev:/dev # for syslog (/dev/log)
+    - /run:/run:ro # for the zedbox pid file
+    - /sys/fs/cgroup/memory/:/sys/fs/cgroup/memory/ # to read and modify memory limits
+    - /proc:/proc #:ro to read memory usage in /proc/<PID> and RW for the eve command (???)
+    - /bin:/bin:ro # for the eve command
+    - /usr/bin:/usr/bin:ro # for the ctr command (it's a dependency of the eve command, bleeugh)
+  pid: host
+  capabilities:
+    - all
```

Comment:
let's break this down and be more granular.

Reply:
Would be nice if you help with it =)

Reply:
So, am I right that to get the proper list, I have to run the tool with all the caps on via `starce` and then - analyze the trace?... Isn't there a tool that can somehow automate it?

Reply:
Ah, ok, there is `capsh`... I'll try to use it.

Reply:
Nope, it looks like it's not suitable... I'm still trying to get the list of necessary caps.

Reply:
while I'm reviewing the code I will write down CAPs you need based on the syscalls.

Reply:
Done. See the last commit I've pushed https://github.com/lf-edge/eve/pull/3984/commits/0bf771ad5d23ef1fa42f876da544115c5d9b135c

Reply:
@shjala, I've done it by disabling all caps and fixing the errors I saw.
But @christoph-zededa proposed another interesting solution!
https://github.com/bpftrace/bpftrace/blob/master/tools/capable.bt
I think it would be nice if we added a tool/doc on how to define a proper list of caps.